packages/cdktf-cli/src/bin/cmds/deploy.ts

```diff
@@ -76,6 +76,12 @@ class Command extends BaseCommand {
         // Setting value to negative will prevent it from being forwarded to terraform as an argument
         default: -1,
       })
+      .option("no-color", {
```

Comment:
We could add this as a note ([chalk docs](https://github.com/chalk/chalk#supportscolor)).
```suggestion
      // The presence of this flag is automatically picked up by `chalk` as well which
      // causes colors to be disabled too for output that the cdktf-cli itself colors.
      .option("no-color", {
```